File: we_13_3_missed_detection_probability_of_a_fault.py
# Integrated Navigation Systems — Chapter 13: Integrity, Fault Detection, and ARAIM
# Worked Example 13.3 — Missed-detection probability of a fault
# Companion code (educational reference implementation; see the book for context).

from math import erf, sqrt
Phi=lambda x:0.5*(1+erf(x/sqrt(2)))
print(f'Pmd={Phi(3.24-6.0):.4f}')   # 0.0029

File: we_13_3_missed_detection_probability_of_a_fault.m
% Integrated Navigation Systems — Chapter 13: Integrity, Fault Detection, and ARAIM
% Worked Example 13.3 — Missed-detection probability of a fault
% Companion code (educational reference implementation; see the book for context).

Td=3.24; b_sigma=6.0; Pmd=normcdf(Td-b_sigma);
fprintf('Pmd=%.4f\n', Pmd);   % 0.0029
